vita3k/mem/src/mem.cpp

```diff
@@ -43,7 +43,14 @@ static void delete_memory(uint8_t *memory) {
 }
 
 static void alloc_inner(MemState &state, Address address, size_t page_count, Allocated::iterator block, const char *name) {
-    uint8_t *const memory = &state.memory[address];
+    Address alignment;
```

Comment:
I think the address should already be aligned, so this should have no effect.

Reply:
Nope :/

Some elfs like "Alone with You" seem to have an odd `p_vaddr` value that are not aligned to 4096.